gettsim/transfers/wohngeld.py

```diff
@@ -63,68 +66,66 @@ def wohngeld_abzüge_m_tu(
 
     """
     abzug_stufen = (
-        (eink_st_tu > 0) + (ges_rentenv_beitr_m_tu > 0) + (ges_krankenv_beitr_m_tu > 0)
+        (eink_st_tu > 0) + (ges_rentenv_beitr_m > 0) + (ges_krankenv_beitr_m > 0)
```

Comment:
Seems okay. Just to be sure: Is health insurance also counted if a pensioner is applying for wohngeld?

Reply:
That's how I would interpret it ([§ 16 WoGG](https://www.gesetze-im-internet.de/wogg/__16.html)), since pensioners either pay Pflichtbeiträge or pay into a private/voluntary insurance plan that serves the same purpose. (also see [here](https://www.deutsche-rentenversicherung.de/DRV/DE/Rente/In-der-Rente/Kranken-und-Pflegeversicherung-der-Rentner/kranken-und-pflegeversicherung-der-rentner.html))